packages/manager/modules/pci/src/projects/project/data-processing/notebooks/delete-notebook/delete-notebook.routing.js

```diff
@@ -0,0 +1,26 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.data-processing.notebooks.delete',
+    {
+      url: '/delete',
+      views: {
+        modal: {
+          component: 'pciProjectDataProcessingQuickDeleteNotebookModal',
+        },
+      },
+      layout: 'modal',
+      params: {
+        notebookId: null,
+        notebookName: null,
+        projectId: null,
+      },
+      resolve: {
+        notebookId: /* @ngInject */ ($transition$) =>
+          $transition$.params().notebookId,
+        notebookName: /* @ngInject */ ($transition$) =>
+          $transition$.params().notebookName,
```

Comment:
**nitpick:** Seems `notebookName` is not used. Could you please remove it?
Also, unless the param is placed in URL, we can not have consistent behavior and this stateParam will be undefined on refresh.

Reply:
Fixed :)